content/en/docs/concepts/configuration/secret.md

```diff
@@ -314,18 +301,20 @@ The output is similar to:
 1f2d1e2e67df
 ```
 
-The program in a container is responsible for reading the secrets from the
-files.
+The program in a container is responsible for reading the secret data from these
```

Comment:
Wording: "The program" to "The application"?

Reply:
Is “program” wrong? It makes sense to me.

Reply:
The program in a container ...
I'd have to reread again. Which program?

Reply:
A running container always has one main process (PID 1 on POSIX), and has 0 or more other processes (typically zero). Each process is running program code.